fetch.bs

```diff
@@ -1782,11 +1806,8 @@ from a {{ReadableStream}} object with <var>reader</var>, run these steps:
 </ol>
 
 <p>To <dfn export for=ReadableStream id=concept-cancel-readablestream>cancel</dfn> a
-{{ReadableStream}} object with <var>reader</var> and <var>reason</var>, return the result of calling
-<a abstract-op>ReadableStreamCancel</a>(<var>reader</var>, <var>reason</var>).
-
-<p class="note no-backref">Because the reader grants exclusive access, the actual mechanism of how
```

Comment:
Why remove this?

Reply:
I thought it was redundant since `ReadableStreamCancel` takes a stream, not a readable

Reply:
I don't understand the connection between those two statements.

The thing you deleted is a general note about readable stream reading, and not related to canceling.

Reply:
Ah ok, I'll add it back